src/sc2laddercore/ToolsUnix.cpp

```diff
@@ -57,7 +57,7 @@ void StartBotProcess(const BotConfig &Agent, const std::string &CommandLine, uns
 
         // NOTE (alkurbatov): For the Python bots we need to search in the PATH
         // for the interpreter.
-        if (Agent.Type == Python)
+        if (Agent.Type == Python || Agent.Type == Wine || Agent.Type == Mono || Agent.Type == DotNetLinux || Agent.Type == CommandCenterLinux)
```

Comment:
Please change DotNetLinux to DotNet

The C# bots will need to be launched with 'dotnet botname.dll' on windows as well because (as I've learned now) there is no exe-binary anymore.

Reply:
@alkurbatov Depends on whether we want the standard action to be `execvp()` or `execv()`?
@m1ndgames will do.

Reply:
We want to use execvp for the cases when we need an additional tool to launch a bit (e.g. python, wine, mono). For others there is no need to enable search in path.

So looks like we need execve for BinaryCpp and CommandCenter only, i.e. we can make much shorter condition.

Reply:
That is style changes to make the code shorter, but it is completely optional.